packages/xrpl/src/sugar/submit.ts

```diff
@@ -52,12 +52,73 @@ async function submit(
  * validated ledger (or has errored/will not be included for some reason).
  * See [Reliable Transaction Submission](https://xrpl.org/reliable-transaction-submission.html).
  *
+ * @example
+ *
+ * ```ts
+ * const { Client, Wallet } = require('xrpl')
+ * const client = new Client('wss://s.altnet.rippletest.net:51233')
+ *
+ * async function submitTransaction() {
+ *   const senderWallet = Wallet.generate()
+ *   const recipientWallet = Wallet.generate()
+ *
+ *   const transaction = {
+ *     TransactionType: 'Payment',
+ *     Account: senderWallet.publicKey,
+ *     Destination: recipientWallet.publicKey,
+ *     Amount: '10'
+ *   }
+ *
+ *   try {
+ *     await client.autofill(transaction)
+ *     const signedTransaction = await Wallet.sign(transaction)
+ *     const result = await client.submitAndWait(signedTransaction)
```

Comment:
I'd recommend replacing this with the 1-line syntax because that's the recommended way to use this library imo. 
```suggestion
 *     await client.submitAndWait(signedTransaction, { wallet: senderWallet })
```

Reply:
(This will also cause a change in the below explanation of what this code does)